crates/nostr/src/nips/mod.rs

```diff
@@ -19,3 +19,5 @@ pub mod nip13;
 pub mod nip19;
 #[cfg(feature = "nip26")]
 pub mod nip26;
+#[cfg(feature = "nip65")]
```

Comment:
Currently the `nip65` module depends only on `base` feature so I think you can set this as `#[cfg(all(feature = "nip65", feature = "base"))]` and remove the `#[cfg(feature = "base")]` inside the module.